### PR TITLE
RTL the desktop site header better (bug 1169738)

### PR DESCRIPTION
**Pull request by spasovski**

### CODE REVIEW

**Comment by sherlockedci** [comment]
[ The Adventure of the Sussex Build](http://sherlocked.dev.mozaws.net/builds/65887304)

File: src/media/css/header.styl [hunk 1]
```diff
@@ -16,7 +16,12 @@ mkt-header-nav {
     order: 3;
 
     +media--wide-tablet() {
-        margin-right: 60px;
+        +ltr() {
+            margin-right: 60px;
+        }
+        +rtl() {
+            margin-left: 60px;
+        }
```
Comment (chuckharmston) [inline src/media/css/header.styl]: This pattern is so :fire: .

File: src/media/css/header.styl [hunk 2]
```diff
@@ -16,7 +16,12 @@ mkt-header-nav {
     order: 3;
 
     +media--wide-tablet() {
-        margin-right: 60px;
+        +ltr() {
+            margin-right: 60px;
```
Comment (chuckharmston) [inline src/media/css/header.styl]: Why not leave this at the top, like the `-`ed L19?

Comment (ngokevin) [inline src/media/css/header.styl]: more explicit that it's LTR only

Comment (chuckharmston) [inline src/media/css/header.styl]: I guess, though it also adds needless specificity.

Comment (mstriemer) [inline src/media/css/header.styl]: I'd rather these things have even margins than special case them for RTL/LTR. The header has 40px of padding but we can give it 35px by removing `padding: 0 40px` (it then falls back to `padding: 0 5px`) from it and setting `margin: 0 30px` on the search toggle, settings toggle and logo. This works fine though.

screenshot 2015-06-11 09 52 11
> Current

screenshot 2015-06-11 09 49 11
> Updated mock up on dev

**Comment by mstriemer** [comment]
If you rebase with master you should get RTL screenshots now.

**Comment by sherlockedci** [comment]
[ The Adventure of the Wisteria Build](http://sherlocked.dev.mozaws.net/builds/66565736)

**Comment by sherlockedci** [comment]
[ The Adventure of the Resident Build](http://sherlocked.dev.mozaws.net/builds/66569360)

**Pull request merged**